src/commands/stepfunctions/README.md

```diff
@@ -29,11 +29,12 @@ datadog-ci stepfunctions uninstrument --step-function <step-function-arn> --forw
-| `--forwarder`     |           | The ARN of the [Datadog Forwarder](https://docs.datadoghq.com/logs/guide/forwarder/) to subscribe Step Function log groups.                                                                                                                               |         |
-| `--env`           | `-e`      | Separate your staging, development, and production environments by `env`. Learn more about [Serverless Tagging](https://docs.datadoghq.com/serverless/guide/serverless_tagging/#the-env-tag). ** **Optional if env tag is already set on Step Functions** |         |
-| `--service`       |           | Group Step Functions belonging to similar workloads by `service`. Learn more about [Serverless Tagging](https://docs.datadoghq.com/serverless/guide/serverless_tagging/#the-service-tag).                                                                 |         |
-| `--dry-run`       | `-d`      | Preview changes without applying them.                                                                                                                                                                                                                    | `false` |
+| `--step-function` | `-s`           | The ARN of the Step Function to be instrumented.                                                                                                                                                                                                          |         |
+| `--forwarder`     |                | The ARN of the [Datadog Forwarder](https://docs.datadoghq.com/logs/guide/forwarder/) to subscribe Step Function log groups.                                                                                                                               |         |
+| `--env`           | `-e`           | Separate your staging, development, and production environments by `env`. Learn more about [Serverless Tagging](https://docs.datadoghq.com/serverless/guide/serverless_tagging/#the-env-tag). ** **Optional if env tag is already set on Step Functions** |         |
+| `--service`       |                | Group Step Functions belonging to similar workloads by `service`. Learn more about [Serverless Tagging](https://docs.datadoghq.com/serverless/guide/serverless_tagging/#the-service-tag).                                                                 |         |
+| `--dry-run`       | `-d`           | Preview changes without applying them.                                                                                                                                                                                                                    | `false` |
+| `--propagate-upstream-trace`       |             | Update Step Function definitions to inject context into invocations to other Step Functions or Lambda functions.                                                                                                                            | `false` |
```

Comment:
is it `--propagate-trace-context` or `--propagate-upstream-trace`?

Reply:
Good catch. I copied that out of old notes